lib/websocket.go

```diff
@@ -29,7 +62,11 @@ var _ WebsocketHandler = (*wsHandler)(nil)
 // can connect to in order to get realtime events
 func NewWebsocketHandler(ctx context.Context, inst *Instance) (WebsocketHandler, error) {
```

Comment:
making this a subpackage means we can't accept an `*Instance` argument, and that's a good thing. Accept a keystore and bus instead.